### DISCUSSION TITLE
How to create test suite with allure and jest-circus?

### DISCUSSION THREAD

**Discussion by yaroslavchaadaev** [allure-framework/allure2#2156] [Questions & Support] [answered]
Hello! Could you help me again please? :)

I'm using JavaScript, Jest, `reporters: ['default']`, `testRunner: 'jest-circus/runner'` for my tests.

Use this code and export in Allure TestOps:

```
describe.only('Suite в Jest', () => {
  it('Проверка 1', async () => {
    expect(2).toBe(2)
  })
  it('Проверка 2', async () => {
    expect(3).toBe(3)
  })
})
```

I want to see in Allure TestOps suite "Suite в Jest", but without success.

Can you please tell me how to correclty create a test suite so that it appears in Allure TestOps? Sorry for that simple question...
![Suite](https://github.com/allure-framework/allure2/assets/89546981/c6d54b95-2c59-4fce-89cc-3bd88770f3e4)

**Comment by yaroslavchaadaev** [comment]


**Reply by baev** [reply]
Please create a separate discussion

**Reply by yaroslavchaadaev** [reply]
Done. And what about my main question? 🙏🏻

**Comment by epszaw** [comment, marked as answer]
It should work by default. Have you configured trees with all the possible suite labels (suite, sub suite, parent suite)?

Try to set up trees in the following way:

![telegram-cloud-document-2-5318898258259491374](https://github.com/allure-framework/allure2/assets/13414205/2140a8ff-def2-486a-a748-c386ad02c02e)

**Reply by yaroslavchaadaev** [reply]
I changed my Tree in Allure TestOps and tried again – the result is the same...

<img width="306" alt="image" src="https://github.com/allure-framework/allure2/assets/89546981/b42c28a4-5adf-41a7-a4b4-0aaba9154327">
<img width="643" alt="image" src="https://github.com/allure-framework/allure2/assets/89546981/e4f9d98f-a3dc-43be-95dc-70bd030bf99b">

**Reply by yaroslavchaadaev** [reply]
I haven't set up a custom field schema. No more questions, thank you.
